public/main.js

```diff
@@ -140,7 +148,7 @@ app.whenReady().then(async () => {
 
   const mainWindow = createMainWindow();
   const displayWindow = createDisplayWindow();
-  displayWindow.setAlwaysOnTop(true, "floating", 1);
+  displayWindow.onAlwaysOnTop(settings.display.alwaysOnTop);
```

Comment:
Please keep the parameters `floating` (status) and `1` (relativeLevel) when the `alwaysOnTop`) is ``true`.
When is `false` just pass the flag not other parameters needed.
https://www.electronjs.org/docs/latest/api/browser-window#winsetalwaysontopflag-level-relativelevel

Reply:
👋 this was a new method ([onAlwaysOnTop](https://github.com/lvcabral/carabiner/pull/1/commits/bcf82eea7742f35e29c81150647984a9a49fe5b8#diff-bea8b3046e769bb45b7b3df65d7b80dde75f65af36516f2a894ede0c97c2b978R55-R62)) that continue to pass the params. 

I see that isn't best practice/maintainable, so I've opted to move it into a helper in 140fc260f06607ac32545623f5640bdc63f378d4